src/services/localStorage.ts

```diff
@@ -76,8 +78,52 @@ export const saveData = async (remoteData: any) => {
   ) {
     const { institution, roles, technologies } = remoteData;
 
+    let orderedRoles: Role[] = roles.map(function parser(value: unknown) {
+      return (value as unknown) as Role;
+    });
+    let orderedTechnologies: Technology[] = technologies.map(function parser(
+      value: unknown
+    ) {
+      return (value as unknown) as Technology;
+    });
```

Comment:
Mesma coisa: dá pra fazer casts simples:
```suggestion
    let orderedRoles = roles as Role[];
    let orderedTechnologies - technologies as Technology[];
```